imod/protocols/protocol_base.py

```diff
@@ -153,6 +153,30 @@ def convertInputStep(self, tsObjId, generateAngleFile=True,
                 self.info("Interpolating tilt series %s with imod" % tsId)
                 Plugin.runImod(self, 'newstack', argsAlignment % paramsAlignment)
 
+                if oddEven:
+
+                    firstItem = ts.getFirstItem()
+                    fnOdd = firstItem.getOdd().split('@')[1]
+                    fnEven = firstItem.getEven().split('@')[1]
+
+                    outputOddTsFileName = os.path.join(tmpPrefix, tsId+EXT_MRCS_TS_EVEN_NAME)
```

Comment:
Maybe this can be refactored in a single method receiving EXT_MRCS_TS_EVEN_NAME or the ODD one?